packages/truffle-compile/package.json

```diff
@@ -13,11 +13,11 @@
     "request-promise": "^4.2.2",
     "require-from-string": "^2.0.2",
     "semver": "^5.6.0",
-    "solc": "0.5.0",
     "truffle-config": "^1.1.12",
     "truffle-contract-sources": "^0.1.3",
     "truffle-error": "^0.0.4",
-    "truffle-expect": "^0.0.8"
+    "truffle-expect": "^0.0.8",
+    "truffle-solc-wrapper": "https://github.com/trufflesuite/truffle-solc-wrapper"
```

Comment:
Let's open an issue at solc-js to start the conversation about them maintaining this.

Reply:
Ah I am out of touch. There is already https://github.com/ethereum/solc-js/issues/360